plugins/chat/assets/javascripts/discourse/components/chat/thread/header.gjs

```diff
@@ -53,6 +53,12 @@ export default class ChatThreadHeader extends Component {
     return this.channel?.threadsManager?.unreadThreadCount;
   }
 
+  get showThreadUnreadIndicator() {
```

Comment:
I added this as it only makes sense if accessing via a channel's thread list or embedded thread (since the unreads are for the thread's channel). If we access via My Threads this doesn't make much sense and is confusing so I've disabled it when the previous route is `chat.channel.threads`.